### PR TITLE
Completed task six, with one 400, one 200 and one 404.

### PR DESCRIPTION
**Pull request by Danmoham**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -174,4 +174,38 @@ describe('Checks all Articles for the NC news',() =>{
...
+        .get('/api/articles/NaN/comments')
+        .expect(400).then((response) =>{
+            expect(response.body.msg).toBe('URL does not exist, the key you gave is not a number - Bad Request!')
+        })
+    })
+    test('Expect 404 not found error for no comments',() =>{
+        return request(app)
+        .get('/api/articles/4/comments')
+        .expect(404).then((response) =>{
+            expect(response.body.msg).toBe('This article has no comments')
```
Comment (robert112233) [inline __tests__/app.test.js:208]: An error implies the client has done something wrong, however if there are no comments for a given article, that's absolutely fine, this should respond with a  200 and an empty array :)

Comment (robert112233) [reply __tests__/app.test.js:208]: You should also include a 404 test for article id = 99999

**Review by robert112233 — CHANGES_REQUESTED** [review]
See feedback and submit another PR for review, you may need to make use of a checkExists function first, before being sure of _why_ you're getting an empty array back. 

Message me if you need clarification ! 

Great work other wise :)

**Review by Danmoham — COMMENTED** [review]
To be changed

**Pull request merged**